webapp/views/App/JobMonitor/InnerJobs/InnerJob/InnerJob.js

```diff
@@ -0,0 +1,39 @@
+import React, { useEffect, useRef } from 'react'
+import PropTypes from 'prop-types'
+
+import * as JobSerialized from '@common/job/jobSerialized'
+
+import { useI18n } from '@webapp/store/system'
+
+import JobProgress from '../../JobProgress'
+import JobErrors from '../../JobErrors'
+
+const InnerJob = ({ innerJob, i }) => {
+  const i18n = useI18n()
+  const elementRef = useRef(null)
+
+  useEffect(() => {
+    if (JobSerialized.getStatus(innerJob) === 'running' && elementRef.current) {
```

Comment:
`JobSerialized.isRunning(innerJob)` can be used instead

Reply:
ouuu i will change!